src/FSharp.Data.GraphQL.Server/Planning.fs

```diff
@@ -208,7 +208,7 @@ let rec private plan (ctx : PlanningContext) (stage : PlanningStage) : PlanningS
         let inner, deferredFields', path' = plan ctx ({ info with ParentDef = info.ReturnDef; ReturnDef = downcast returnDef; Identifier = "__index" }, deferredFields, "__index"::info.Identifier::path)
         { info with Kind = ResolveCollection inner }, deferredFields', path'
     | Abstract _ -> 
-        planAbstraction ctx info.Ast.SelectionSet (info, deferredFields, path) (ref []) None 
+        planAbstraction ctx info.Ast.SelectionSet (info, deferredFields, info.Identifier::path) (ref []) None 
```

Comment:
This is the root of the problem. Interfaces planning wasn't adding its identifier to the path, causing wrong results in the `traversePath` function.